samtranslator/translator/arn_generator.py

```diff
@@ -23,6 +23,7 @@ def _region_to_partition(region: str) -> str:
         "us-isob": "aws-iso-b",
         "us-gov": "aws-us-gov",
         "eu-isoe": "aws-iso-e",
+        "us-isof": "aws-iso-f",
```

Comment:
not really sure how it works but do we need to remove the use of `${AWS::Partition}` in this file too?

Reply:
Yea you're right, I'll remove it